packages/falso/src/lib/mask.ts

```diff
@@ -11,12 +11,43 @@ import { fake, FakeOptions, getRandomInRange } from './core/core';
  *
  * @example
  *
+ * randMask({ mask: "@#### @##" })
+ *
+ * @example
+ *
  * randMask({ length: 10 })
  *
  */
-export function randMask<Options extends FakeOptions>(options?: Options) {
-  return fake(
-    () => getRandomInRange({ min: 1000, max: 9999 }).toString(),
-    options
-  );
+export function randMask<
+  Options extends FakeOptions & {
+    mask?: string;
+    char?: string;
+    digit?: string;
+  }
+>(options?: Options) {
+  return fake(() => {
+    const [mask, char, digit] = [
+      options?.mask ?? '@##@#',
+      options?.char ?? '@',
+      options?.digit ?? '#',
+    ];
+
+    const randomChars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+    const randomDigits = '0123456789';
+
+    return mask
+      .split('')
+      .map((item) => {
+        if (item == char)
+          return randomChars.charAt(
+            getRandomInRange({ min: 0, max: randomChars.length })
+          );
+        else if (item == digit)
+          return randomDigits.charAt(
```

Comment:
This could just us `getRandomInRange({ min: 0, max: 9, fraction: 0 })` as you are already calling this function